Userland/Services/WebContent/WebContentServer.ipc

```diff
@@ -78,6 +78,7 @@ endpoint WebContentServer
     alert_closed() =|
     confirm_closed(bool accepted) =|
     prompt_closed(Optional<String> response) =|
+    color_picker_closed(Optional<String> response) =|
```

Comment:
Feels a little weird to pass the colour as a String, but maybe that does make sense since that's what the Input's value is stored as. :thinking: It does feel awkward on the browser side though.

Reply:
Can IPC pass Color types?

Reply:
Yup, see https://github.com/SerenityOS/serenity/blob/6931a5a0a8e8b2b9cb4a9acb690b12e97be61ece/Userland/Services/WindowServer/WindowServer.ipc#L149 for example.

I'm not entirely sure, but it seems like dealing with the Color<->String conversion in WebContent, and letting the browser chromes just deal with Colors would be more convenient.

Reply:
Sounds like a good idea